cadl-extension/src/code-model-builder.ts

```diff
@@ -701,7 +701,13 @@ export class CodeModelBuilder {
   }
 
   private processParameterBody(op: CodeModelOperation, body: ModelProperty | Model, parameters: Model) {
-    const schema = this.processSchema(body.kind === "Model" ? body : body.type, body.name);
+    let schema: Schema;
+    if (body.kind === "ModelProperty" && body.type.kind === "Scalar" && body.type.name === "bytes") {
+      //handle for binary body
+      schema = new BinarySchema(body.name);
```

Comment:
Ideally this should be a `processBinary` which will add it to `codeModel.schemas` (that's the side-effect)
https://github.com/Azure/autorest.java/blob/54b1738de6d48b23a298fda4581198c2b3af2815/cadl-extension/src/code-model-builder.ts#L1142-L1148

Recommend to do this.